From b4f5273b9f01d4e0f22c4c1b66396ef300065f10 Mon Sep 17 00:00:00 2001
From: Noam Lessner <nlessner@habana.ai>
Date: Tue, 17 Dec 2019 09:57:44 +0200
Subject: [PATCH] [SW-6739] arcboot: libsb: add arcv2 optimization

add compiler flag to optimize the secureboot library from
synopsys for ARCv2

Change-Id: I3d94c8af165de8f9c2356d587ece1d06fdf3e79f
Signed-off-by: Noam Lessner <nlessner@habana.ai>
---

diff --git a/Makefile b/Makefile
index 3c79a40..c2dbb83 100644
--- a/Makefile
+++ b/Makefile
@@ -29,7 +29,8 @@
 
 # This combination allows removing unused symbols from libsb.a and libellipsys.a
 LDFLAGS += -Wl,--gc-sections
-export CFLAGS = -Os -fdata-sections -ffunction-sections
+CFLAGS = -DELP_ARCV2
+export CFLAGS += -Os -fdata-sections -ffunction-sections
 
 ARCBOOT_SRCS_C := common/arcboot.c
 ARCBOOT_SRCS_S := common/crt0.S
